interfaces/builtin/desktop_launch_test.go

```diff
@@ -101,3 +104,27 @@ func (s *desktopLaunchSuite) TestStaticInfo(c *C) {
 	c.Assert(si.BaseDeclarationPlugs, testutil.Contains, "deny-auto-connection: true")
 	c.Assert(si.BaseDeclarationPlugs, testutil.Contains, "allow-installation: false")
 }
+
+func (s *desktopLaunchSuite) TestDesktopLaunchAndDesktopLegacy(c *C) {
+	const desktopLaunchConsumerYamlWithLaunchAndLegacy = `
+name: other
+version: 0
+apps:
+ app:
+    command: foo
+    plugs: [desktop-launch, desktop-legacy]
+`
+
+	plug, _ := MockConnectedPlug(c, desktopLaunchConsumerYamlWithLaunchAndLegacy, nil, "desktop-launch")
+	slot, _ := MockConnectedSlot(c, desktopLaunchCoreYaml, nil, "desktop-launch")
+	appSet, err := interfaces.NewSnapAppSet(s.plug.Snap(), nil)
+	c.Assert(err, IsNil)
+	apparmorSpec := apparmor.NewSpecification(appSet)
+	err = apparmorSpec.AddConnectedPlug(s.iface, s.plug, s.slot)
+	c.Assert(err, IsNil)
+	err = apparmorSpec.AddConnectedPlug(builtin.MustInterface("desktop-legacy"), plug, slot)
+	c.Assert(err, IsNil)
+	c.Assert(strings.Contains(apparmorSpec.SnippetForTag("snap.other.app"), "# Explicitly deny access to other snap's desktop files"), check.Equals, false)
```

Comment:
I think this could use Not(testutil.Contains) ?

Reply:
Yesss!!! That is what I was trying to do. Changed. Thanks!